OracleSiebel/OEL/build/multi-stage/cache/dockerfile

```diff
@@ -0,0 +1,81 @@
+# Globals
+
+ARG VERSION=19.03
+ARG BUILD_DATE=2019-04-30
+
+
+FROM oraclelinux:latest as temp
+ARG INSTALLSITE=installers
+ARG VERSION
+
+COPY oraInst.loc getBase.sh getUpdate.sh cache/cache_base.rsp cache/cache_install_base.sh cache/cache_update.rsp.templ cache/cache_install_update.sh siebelkeystore.jks /config/
+
+# Install Siebel Base
+
+ENV LANG=en_US.UTF-8
+
+RUN mkdir -p -ma+rwx /siebel/cache /siebel/oraInventory /stage \
+&& groupadd -g 1000 siebel && useradd -m -g siebel -G siebel -u 1000 siebel \
+&& yum -y install wget tcsh ksh \
+&& cd /mnt \
+&& date \
+&& echo Downloading installer \
+&& bash /config/getBase.sh ${INSTALLSITE} \
+&& date \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/unzip \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/runInstaller \
+&& cd .. \
+&& echo Installing Siebel ... \
+&& bash /config/cache_install_base.sh \
+&& echo Waiting for deployment of web archive ... \
+&& sleep 30 \
+&& mkdir /siebel/cache/ps_backup/ \
+&& chown -R siebel:siebel /siebel \
+&& chmod -R 755 /siebel/cache
+
+# Install Siebel Update
+
+RUN cd /mnt \
+&& date \
+&& echo Downloading installer \
+&& bash /config/getUpdate.sh ${INSTALLSITE} ${VERSION} \
+&& date \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/unzip \
+&& chmod +x /mnt/Siebel_Enterprise_Server/Disk1/install/runInstaller \
+&& cd .. \
+&& bash /config/cache_install_update.sh \
+&& echo Waiting for deployment of web archive ... \
+&& sleep 30 \
+&& rm -rf /mnt/Siebel_Enterprise_Server/ \
+&& rm -rf /siebel/cache/ps_backup/ \
+&& chmod -R 755 /siebel/cache
+
+# Create Final Container
+
+FROM oraclelinux:latest
```

Comment:
Please switch to `oraclelinux:7-slim` or at least `oraclelinux:7` as `oraclelinux:latest` will switch to 8 fairly soon

Reply:
Done